### PR TITLE
Add view commands for products and clients

### PR DESCRIPTION
**Pull request by benluiwj**
haven't integrated fully into address book and no testing done yet

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewClientCommand.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ * Views a client based on the id specified.
+ */
+public class ViewClientCommand extends Command {
+    public static final String COMMAND_WORD = "view -c";
+
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Views a current client identified by the index number used in the displayed "
+                    + "product list.\n"
```
Comment (SkyBlaise99) [inline src/main/java/seedu/address/logic/commands/ViewClientCommand.java:18]: Change to `Client list`?

File: src/main/java/seedu/address/logic/commands/ViewProductCommand.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Views a product based on the index specified.
+ */
+public class ViewProductCommand extends Command {
+    public static final String COMMAND_WORD = "view -p";
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Views a current product identified by the index number used in the displayed "
+                    + "product list.\n"
+                    + "If no such client exists, nothing will be shown\n"
```
Comment (SkyBlaise99) [inline src/main/java/seedu/address/logic/commands/ViewProductCommand.java:18]: change to `product`

File: src/main/java/seedu/address/logic/commands/ViewProductCommand.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                    + "Parameters: INDEX (must be a positive integer)"
+                    + "Example usage : "
+                    + COMMAND_WORD
+                    + " 20 ";
+    private final ProductContainsIdPredicate predicate;
+    ViewProductCommand(ProductContainsIdPredicate predicate) {
+        this.predicate = predicate;
+    }
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```
Comment (SkyBlaise99) [inline src/main/java/seedu/address/logic/commands/ViewProductCommand.java:28]: Add newlines pls

File: src/main/java/seedu/address/model/product/ProductContainsIdPredicate.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.product;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Client}'s {@code id} matches the id given.
```
Comment (SkyBlaise99) [inline src/main/java/seedu/address/model/product/ProductContainsIdPredicate.java:9]: change to `product`

**Review by SkyBlaise99 — CHANGES_REQUESTED** [review]
some small stuffs needs update then should be ok to merge
also update the switch case in addressbookparser so the app actually run ur code

File: src/main/java/seedu/address/logic/commands/ViewProductCommand.java (hunk 3 above)
Comment (benluiwj) [reply src/main/java/seedu/address/logic/commands/ViewProductCommand.java:28]: check style failed when I new lined but ill include it as a change anyways

Comment (SkyBlaise99) [reply src/main/java/seedu/address/logic/commands/ViewProductCommand.java]: lol wut and why haha

**Pull request merged**